frontend/src/app/features/work-packages/components/filters/filter-searchable-multiselect-value/filter-searchable-multiselect-value.component.ts

```diff
@@ -79,14 +78,15 @@ export class FilterSearchableMultiselectValueComponent extends UntilDestroyedMix
 
   @ViewChild('ngSelectInstance', { static: true }) ngSelectInstance:NgSelectComponent;
 
-  constructor(readonly halResourceService:HalResourceService,
+  constructor(
+    readonly halResourceService:HalResourceService,
     readonly apiV3Service:ApiV3Service,
     readonly cdRef:ChangeDetectorRef,
     readonly I18n:I18nService,
     protected currentProject:CurrentProjectService,
     protected currentUser:CurrentUserService,
     readonly halNotification:HalResourceNotificationService,
-    readonly ngZone:NgZone) {
```

Comment:
I suppose this is not a typo but really no longer needed? :sweat_smile:

Reply:
It was unused, so I removed it ;)